### PR TITLE
feat: Add how-to re creating a new cloud account

### PR DESCRIPTION
**Pull request by colder-is-better**
We welcome new users to the Cleura Cloud by showing how one can create a new account in the Cleura OpenStack Dashboard.

### CODE REVIEW

File: docs/howto/openstack/create-new-account/cleura-cloud.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+* __Step 3 - Account verification__. While the new account is being created, and before it becomes fully operational, you have to take one last step toward verification. You do that either by entering valid credit card information or by placing a simple phone call. Should you choose to verify by credit card, rest assured that no charge will take place -- no money will be drawn from the card, in other words. On the other hand, if you prefer to verify by phone, you may certainly do so during business hours (08:00-17:00 CET). If you choose to call, please remember that you will be asked for the username of the new account, so have that piece of info handy.
```
Comment (fghaas) [inline docs/howto/openstack/create-new-account/cleura-cloud.md]: I know I am being an extreme stickler here, but the punctuation before "no money" needs to be an em dash (`---`), not an en dash (`--`). :)

File: docs/howto/openstack/create-new-account/cleura-cloud.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+After the account verification is complete, you are greeted by the Cleura Cloud Control Panel. Feel free to follow through the introductory guide to the environment --that will not take long-- or skip it and start taking advantage of the Cleura Cloud without delay.
```
Comment (fghaas) [inline docs/howto/openstack/create-new-account/cleura-cloud.md]: Here, too, we need em dashes.

**Review by fghaas — COMMENTED** [review]
In terms of content, this is excellent. Kudos!

It got me thinking on where in the documentation we want this information though, because a Control Panel account is of course necessary for everything: OpenStack, Kubernetes, object storage. So, I think it makes more sense if we create a "Getting Started" section under the How-to guides, and move the page there.

Thus I have pushed 86ba3d674a6b711f84873cb7a92defc12844b5c1 to your branch. If you are happy with that change, you can pull from your branch into your local checkout, run `git rebase --interactive --autosquash main`, squash my commit into yours, and then force-push back to your branch.

Other than that, I have just a few small nits. And, do we want to add screenshots of the sign-up process?

**Review by colder-is-better — COMMENTED** [review]
I agree we need a new "Getting Started" section, and I will also act upon the rest of the comments. Regarding screenshots, though, I believe those won't be necessary. The whole sign-up process is pretty fast and straightforward, and the UI is clean and simple enough. There's no room for misunderstanding or error, I believe.

**Review by fghaas — COMMENTED** [review]
Couple more notes.

**Review by fghaas — COMMENTED** [review]
Please crop the screenshots so that chrome (address bars, title bars etc.) that is specific to individual browsers, operating systems, and desktop environments is removed, and we only show the content that is expected to be identical in any browser using the given resolution.

See example in fixup commit 15efdd1bd97104b41496cc1c7ac07ce1095c070f. Thanks!

**Comment by colder-is-better** [comment]
OK, screenshots should be OK now -- thanks!

**Comment by fghaas** [comment]
The screenshots look fine by themselves now, but they obviously still need to be included in the text.

**Comment by colder-is-better** [comment]
> The screenshots look fine by themselves now, but they obviously still need to be included in the text.

I pushed an older version of the document by mistake, but it should be OK now.

**Comment by fghaas** [comment]
This looks good to me now, with two suggestions:

* Within the "Getting Started" section, I think this should be a top-level document rather than one that heads its own subsection. See d0aa02832f9b5861023970eb912271d8bf67416b for a suggested fixup commit.
* For the screenshots, I also propose we include b7a5feba2f2813266c54d7782dc7b0e9554f5438 so as to be able to zoom and cycle through them.

**Comment by colder-is-better** [comment]
> This looks good to me now, with two suggestions:
> 
>     * Within the "Getting Started" section, I think this should be a top-level document, rather than one that heads its own subsection. See [d0aa028](https://github.com/citynetwork/docs/commit/d0aa02832f9b5861023970eb912271d8bf67416b) for a suggested fixup commit.
>

Simpler/cleaner structure, nice!
 
>     * For the screenshots, I also propose we include [b7a5feb](https://github.com/citynetwork/docs/commit/b7a5feba2f2813266c54d7782dc7b0e9554f5438), so as to be able to zoom and cycle through them.

Very useful plugin!

**Comment by fghaas** [comment]
Looks like on your last update you squashed not only the fixup commit, but the separate one that modifies `tox.ini` and `mkdocs.yml`. I've taken the liberty to undo that, and force-push one more time (simultaneously rebasing this change on the current state of `main`).

**Comment by fghaas** [comment]
@colder-is-better I'd say this is now good to merge. Any objections?

**Review by fghaas — APPROVED** [review]

**Comment by colder-is-better** [comment]
> @colder-is-better I'd say this is now good to merge. Any objections?

Nothing I can think of. I think it's time :)

**Pull request merged**